firefox-ios/firefox-ios-tests/Tests/XCUITests/FxScreenGraph.swift

```diff
@@ -1085,7 +1088,7 @@ func createScreenGraph(for test: XCTestCase, with app: XCUIApplication) -> MMScr
     }
 
     map.addScreenState(BrowserTabMenu) { screenState in
-        sleep(1)
+        sleep(3)
```

Comment:
I would not add this... have you tried the nowAt(BrowserTabMenu) in the test instead?
This will increase a lot the tests for all test suites not only for those that run in the github actions